vars/perfRegressionParallelPipeline.groovy

```diff
@@ -115,6 +115,7 @@ def call(Map pipelineParams) {
                                                         export SCT_INSTANCE_PROVISION=${params.provision_type}
                                                         export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$GIT_BRANCH | sed -E 's+(origin/|origin/branch-)++')
                                                         export SCT_AMI_ID_DB_SCYLLA_DESC=\$(echo \$SCT_AMI_ID_DB_SCYLLA_DESC | tr ._ - | cut -c1-8 )
+                                                        export SCT_STORE_RESULTS_IN_ELASTICSEARCH=true
```

Comment:
this will make it always `True` for all jobs that use this `groovy` code..
isn't it better to set it inside the config files?

i see most (if not all) `.yaml` files related to perf-regression that they already have `store_perf_results: true`

Reply:
That flag is only used once in the code
https://github.com/scylladb/scylla-cluster-tests/blob/73a8c1c024fa0f083cf7fc154f35afbfa909551b/sdcm/db_stats.py#L690

So I'm remove this change completely and gonna leave only one flag.